staff/tere-albenca/git/README.md

```diff
@@ -0,0 +1,23 @@
+# GIT COMANDS
+
+- git status (para ver lo que has hecho)
+- git branch (para crear nueva rama)
+- git add staff/...(para añadir contenido, puedes añadir una carpeta o un solo archivo)
+- git commit -m "lo que has hecho #ISSUE" (para comentar los cambios que has hecho , has de poner #issueNumber)
+- git push (para volcarlo en github)
+- git push -f (se vuelca el contenido pero con los cambios realizados en local, por si elminas cosas)
+
+
+### Para resetear carpetas
+1. git log (se ve todas las versiones que has subido con su commit- copias el identificador bueno)
+2. :q (para salir del log)
+3. git reset (y pegas el identificador del commit bueno + enter)
+
+
```

Comment:
Put similar spaces between similar items

Reply:
Is good now? I don't know what do you want